Sources/Datadog/Core/FeaturesConfiguration.swift

```diff
@@ -140,9 +140,9 @@ extension FeaturesConfiguration {
 }
 
 private func ifValid(environment: String) throws -> String {
-    let regex = #"^[a-zA-Z0-9_]+$"#
+    let regex = #"^[a-zA-Z0-9_./-]+$"#
```

Comment:
[There are more rules](https://docs.datadoghq.com/getting_started/tagging/#defining-tags) to include in this regexp, i.e. _"must start with a letter"_, _"can be up to 200 characters long"_.

Reply:
Ref to Android regexp: https://github.com/DataDog/dd-sdk-android/pull/369#discussion_r486145119